frontend/src/utils/queryGraph.js

```diff
@@ -124,7 +124,7 @@ const convert = {
     reasonerRepresentation.nodes.forEach(pruneCuries);
     reasonerRepresentation.nodes.forEach(pruneTypes);
 
-    reasonerRepresentation.nodes.forEach(removeLabel);
+    // reasonerRepresentation.nodes.forEach(removeLabel);
```

Comment:
If I create a query graph with one node, download it, and then upload it again I get this:

<img width="1167" alt="Screen Shot 2021-01-16 at 1 18 30 PM" src="https://user-images.githubusercontent.com/1634866/104819726-72d77f80-57fd-11eb-9f4b-afa1b93df044.png">

Reply:
I tried this out and it seems to work for me. I tried it with both a curie node and a generic typed node. Could you provide more specific steps to reproduce please?